calculators/election_totals.rb

```diff
@@ -28,8 +28,9 @@ def fetch
           .includes(:office_election, :calculations)
           .find_each do |candidate|
 
-          total = candidate.calculation(:total_contributions)
-          total_contributions += total;
+          total = candidate.calculation(:total_contributions) ||
+            candidate.calculation(:contribution_list_total)
```

Comment:
I don't see where the contribution_list_total calculation is ever saved for the candidate.